src/lib/parameters.ml

```diff
@@ -3,57 +3,151 @@ module String = Sosa.Native_string
 
 
 type t = {
-  (* MHC Alleles which take precedence over those generated by Seq2HLA. *)
-  mhc_alleles: string list option;
-  mouse_run: bool [@default false];
-  with_topiary: bool [@default false];
-  with_seq2hla: bool [@default false];
-  with_mutect2: bool [@default false];
-  with_varscan: bool [@default false];
-  with_somaticsniper: bool [@default false];
-  with_bqsr: bool [@default true];
-  with_indel_realigner: bool [@default true];
-  with_mark_dups: bool [@default true];
-  with_optitype_normal: bool [@default false];
-  with_optitype_tumor: bool [@default false];
-  with_optitype_rna: bool [@default false];
-  with_kallisto: bool [@default false];
-  email_options: Qc.EDSL.email_options option;
-  bedfile: string option [@default None];
+  (* SAMPLES *)
+  normal_inputs: string list; [@docs "SAMPLES"]
+  (** Normal sample(s) for the pipeline. *)
+  tumor_inputs: string list; [@docs "SAMPLES"]
+  (** Tumor sample(s) for the pipeline. *)
+  rna_inputs: string list option; [@docs "SAMPLES"]
+  (** RNA sample(s) for the pipeline. *)
+
+  (* OPTIONS *)
+  dry_run: bool;
+  (** Dry-run; does not submit the pipeline to a Ketrew server. *)
+  output_dot_to_png: string option; [@docv "PATH"]
+  (** Output the pipeline as a PNG file. *)
```

Comment:
Those 2 options are actually not pipeline parameters; there are client-side run-time things; it'd be cleaner to have them in a separate `Command_line.Options.t` and have `Command_line.pipeline` take 1 more argument.

(Epidisco is a library, so somebody calling `Parameters.make` from utop or a script should not have to care about these)